grpc-server-runtime/build.gradle

```diff
@@ -18,8 +18,8 @@ dependencies {
     compileOnly "io.micronaut.discovery:micronaut-discovery-client"
     compileOnly "io.micronaut:micronaut-tracing"
     compileOnly "io.micronaut:micronaut-management"
+    compileOnly "io.micronaut:micronaut-netty-graal"
```

Comment:
This doesn't help. It would need to be `compileOnly` in end user apps

Reply:
I tested locally and just with `implementation("io.micronaut.grpc:micronaut-grpc-runtime")` (which I already had) in my test application it pulled in `micronaut-netty-graal`. See https://github.com/micronaut-graal-tests/micronaut-grpc-graal/blob/3.3.x/server/build.gradle#L26

Reply:
that doesn't make sense to me, is that correct @melix ?

Reply:
If `micronaut-netty-graal` exposes things which should only be present on the consumer's compile classpath (therefore not on the _runtime_ classpath), then the right configuration is `compileOnlyApi`.

Reply:
(and I agree that it doesn't make sense that `compileOnly` would bring `micronaut-netty-graal` transitively, there must be something else bringing it, which a `./gradlew dependencyInsight --configuration compileClasspath --dependency micronaut-netty-graal` should reveal).

Reply:
I'm going to remove all snapshots and everything and try it again. Maybe I missed something because dependencyInsight doesn't return anything.

Reply:
I've double-checked and as both of you mentioned, it doesn't work. I don't know what was going on there. Adding the dependency as `compileOnlyApi` leads to be displayed when running `dependencyInsigth` in the application. The strange thing is that the GraalMV feature is not executed (I've put a `println`) for my grpc-server app but it is for my grpc-client app.

Reply:
Like I said in another issue I still think it is better to just add the changes to `buffer-netty` and add a dependency to that module on this one.